packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -316,9 +357,14 @@ export const moduleSelected: Action<{
       }
 
       effects.live.sendUserCurrentModule(module.shortid);
+
+      if (!state.editor.isInProjectView) {
+        actions.editor.internal.updatePreviewCode();
+      }
     }
   } catch (error) {
-    state.editor.currentModuleShortid = null;
+    // Do nothing, it is most likely VSCode selecting a file
```

Comment:
Haha, I love this text. I do think this is the cause for not unselecting a file when you open a file that's not a module (try CMD+clicking) op `react-dom` from the editor.

Reply:
Hehe, yeah, this is actually changed to:

```
// You jumped to a file not in the Sandbox, for example typings
```
now 😂